CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# _1.90_ (2023-12-01)
+- **(Bug Fix)** Amazon Q Transform: Fix an issue where the IntelliJ may freeze after clicking "Transform"
+
```

Comment:
do `./gradlew newChange` instead because this file is autogenerated

Reply:
Thanks!